persistence/src/test/scala/hmda/persistence/processing/HmdaFileParserSpec.scala

```diff
@@ -35,8 +67,30 @@ class HmdaFileParserSpec extends ActorSpec {
     "persist parsed LARs and parsing errors" in {
       parseLars(badLines)
       probe.send(hmdaFileParser, GetState)
-      probe.expectMsg(HmdaFileParseState(5, Seq(List("Agency Code is not an Integer"))))
+      probe.expectMsg(HmdaFileParseState(2, Seq(List("Agency Code is not an Integer"))))
+    }
+
+    "read entire raw file" in {
+      val hmdaFileParser2 = createHmdaFileParser(system, "12345-2017-2")
+      val hmdaRawFile = createHmdaRawFile(system, "12345-2017-2")
+      for (line <- lines) {
+        probe.send(hmdaRawFile, AddLine(timestamp, line.toString))
+      }
+      probe.send(hmdaRawFile, GetState)
+      probe.expectMsg(HmdaRawFileState(4))
+
+      val msg = "Parsing ended for 12345-2017-2"
+      EventFilter.debug(msg, source = hmdaFileParser2.path.toString, occurrences = 1) intercept {
```

Comment:
I believe it might be better to keep the original test as well as adding this one. Just to make sure that the data has been properly loaded.